airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/client/ClickhouseAirbyteClient.kt

```diff
@@ -101,8 +106,77 @@ class ClickhouseAirbyteClient(
         tableName: TableName,
         columnNameMapping: ColumnNameMapping
```

Comment:
Do we not need to do anything with `columnNameMapping`?

Reply:
I'll check with ed on Monday